db/schema.rb

```diff
@@ -115,6 +115,17 @@
     t.index ["seller_id"], name: "index_menu_items_on_seller_id"
   end
 
+  create_table "open_hours", force: :cascade do |t|
+    t.bigint "seller_id", null: false
+    t.time "open"
+    t.time "close"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.integer "openday"
```

Comment:
nit and you don't have to change it if you don't want to / can do it as a follow up; but could you make this open_day?

Reply:
changed!